test/test_distributions.py

```diff
@@ -519,6 +548,28 @@ def test_log_prob(jax_dist, sp_dist, params, prepend_shape, jit):
     assert_allclose(jit_fn(jax_dist.log_prob)(samples), expected, atol=1e-5)
 
 
+@pytest.mark.parametrize('jax_dist, sp_dist, params', CONTINUOUS)
+def test_cdf_and_icdf(jax_dist, sp_dist, params):
+    if not sp_dist:
+        pytest.skip('no corresponding scipy distn.')
```

Comment:
Are all the `.cdf()` and `.icdf()` methods actually tested? You might want to add some simple algebraic tests that are independent of scipy:
- `d.cdf(d.icdf(u)) = u` for `u ~ uniform(0,1)`
- `d.icdf(d.cdf(x)) = x` for `x ~ d`
- `d/dx d.cdf(x) = d.log_prob(x).exp()` for `x ~ d`
- `d/du d.icdf(u) = 1 / d.log_prob(d.icdf(u)).exp()` for `u ~ uniform(0,1)`

Reply:
Thanks, those tests are nice! I just added them.

> Are all the .cdf() and .icdf() methods actually tested?

Yes, the test exercises all implemented methods.